tests/callbacks/log_plays_coveralls.py.lol

```diff
@@ -0,0 +1,206 @@
+# This file is part of Ansible
```

Comment:
No, this is the start of coveralls for ansible 2.0. I prefer to have it now as a skeleton and update after. Is it better to remove it?